### ISSUE TITLE
TSC Elections Chair and Director - Call for nominations.

### ISSUE THREAD

**Issue by mhdawson** [nodejs/TSC#521]
May is the month for TSC Elections.  The proposed schedule is:

April 16 - Call for nominations
April 30 - Close nominations
May 1 - Open balloting
May 15 - Announce Results
May 16-29- Handover
May 30 (or earlier if handover complete) Transition complete

This issue is to open the call for nominations.

Depending on the number of nominations we'll figure out if we need to have people write up something about why they would be a good fit for the position.

Current status

**Chair Nominees**:

@mhdawson - Michael Dawson
@Trott - Rich Trott

**Director Nominees**:

@MylesBorins - Myles Borins

**Comment by mhdawson** [comment]
@nodejs/tsc

**Comment by Trott** [comment]
Probably best for the incumbents to indicate soon if they are running for re-election or not. Whether or not there is an incumbent running will likely affect other people's decisions on whether to self-nominate or not.

**Comment by MylesBorins** [comment]
I self-nominate to run for Director again.

Jumping between meetings today so I don't have a ton of time to write an extended post at the moment, but will follow up soon with more reasoning as to why I should be considered.

**Comment by mhdawson** [comment]
I self-nominate to run for Chair again.  Also fighting fires today so will add more later on.

**Comment by Trott** [comment]
Can we get agreement at this point if the election will be a secret ballot or not? I have conflicting feelings either way, but I think it would be good to get that decision made now while there's time to discuss it rather than rushing to make it at the last minute. (Disclaimer: I haven't looked at our governance docs to see if they indicate whether it must or must not be a secret ballot.)

**Comment by Trott** [comment]
I self-nominate for Chair. I think Michael has done a commendable job as Chair, so this isn't a protest or anything. I do think it's healthy to have an election with more than one candidate and I hope at least one more person will run for Director for that reason. There is also a benefit to not having the same person in a position for too many consecutive years.

I believe my time facilitating CTC meetings showed that I have a proven track record at effectively organizing and running meetings. I have a strong belief that the TSC Chair position is *not* inherently a leadership position (despite the soft power it necessarily entails) and should not be elevated as a representative of the TSC. (That is more appropriate for the Director position.) I believe the role of the TSC Chair is to make meetings as effective (and unnecessary!) as possible. 

If elected, I will energetically work to resolve as many issues as possible outside of meetings so that the agendas for meetings can be as focused as possible. This was something I did for the CTC and I would be happy to start doing it again for the TSC.

I welcome questions, especially from anyone who might feel unfamiliar with my track record (since I'm basically running on "I did this really well for the CTC and I can do it really well for the TSC").

**Comment by richardlau** [comment]
>  (Disclaimer: I haven't looked at our governance docs to see if they indicate whether it must or must not be a secret ballot.)

I don't know if it's specified elsewhere but it's not specified in the charter:
https://github.com/nodejs/TSC/blob/master/TSC-Charter.md#section-7-elections

(I only read this recently since when this issue was first opened yesterday it was unclear what these elections were for or who could vote in them. For the benefit for anyone else, these elections are for the TSC chair and TSC director, voted for by members of the TSC, as outlined in the TSC charter linked above.)

**Comment by jasnell** [comment]
+1 to secret ballot

**Comment by mhdawson** [comment]
I believe it was secret last time, +1 to secret from me as well.

**Comment by cjihrig** [comment]
+1 to a secret ballot.

**Comment by MylesBorins** [comment]
+1 to secret ballot

**Comment by MylesBorins** [comment]
@mhdawson do you think we should keep the original post updated with current status / nominees?

Text you can copy / paste

**Chair Nominees:**

@mhdawson - Michael Dawson
@jasnell - James M Snell

**Director Nominees:**

@MylesBorins - Myles Borins

**Comment by cjihrig** [comment]
@MylesBorins I think the chair nominations are Michael and Rich, right? Not James.

**Comment by jasnell** [comment]
@cjihrig 
> I think the chair nominations are Michael and Rich, right? Not James.

That is correct. I am *not* running for either position.

**Comment by MylesBorins** [comment]
my apologies, I've updated my comment. This is what I get for trying to get one more thing in before the end of the day. Sorry for putting you in that position @jasnell

**Comment by mhdawson** [comment]
@MylesBorins done.

**Comment by mhdawson** [comment]
Short outline for my reasons for running for Chair:

I'm running as chair again this year for a number of reasons:

* I believe I've done a good job keeping the meetings running, making sure we have minutes
  and getting them landed during my first year (Many thanks to those who have helped chair
  a number of the meetings). Along with that I've also helped to make the
  process more efficient through the auto-generation of the meeting artifacts. 
* I've helped to shape the content of the meetings to try and focus a bit more on larger/longer
  term efforts (strategic efforts), something I'd like to continue to do this year.
* My participation in a broad range of the working groups, teams and initiatives (benchmarking, build,
  security-wg, diagnostics, user-feedback, n-api) help me bring a good insight to the role with respect
  to what is going on outside of core.
* I've enjoyed being pulled in with the Director on some of the foundation efforts and I'm willing and
  enthusiastic to help out when needed on that front. I feel that it is good to
  have more than one person involved on this front as the Chair and the Director
  can work together to provide greater coverage and it is a better representation of our decentralized 
  approach.
* I believe being the Chair of the TSC and actively participating in the Community Committee
  has started to help build the relationship between the two groups and I'd like to be
  able to continue that process this year.

**Comment by mhdawson** [comment]
There has been support (+1s) for making the vote private, and no objections.  I'm going to propose we move forward with the vote being private. I've reached out to @bnb who has agreed to run the voting process for us (so we have somebody independent).  With the current number of nominations, I think a simple vote would be fine.  If we get more we can discuss if we need something more complicated.

**Comment by mhdawson** [comment]
@nodejs/tsc - a reminder that its the last week to nominate.

**Comment by MylesBorins** [comment]
Today is the last day y'all

**Comment by bnb** [comment]
First off, let me be the first to congratulate @MylesBorins on continuing his role as TSC Director 🎉 

I'm setting up the vote now. I know the CommComm is relatively used to the platform that we'll be using, but I wanted to give y'all a quick crash course on how this is going to work.

**TL;DR:** Check your email for a link from me. Open the link and select your choice.

All @nodejs/tsc members will receive their voting link at the email address listed in the [TSC membership section](https://github.com/nodejs/node#tsc-technical-steering-committee) of the nodejs/node `README.md`. I will be sending them individually from my personal email address, `hello@bnb.im`.

**Once you receive your link, do not share it with _anyone_. Your link is your unique vote.**

Please vote as soon as possible! The sooner everyone votes, the sooner the Chair role will be decided. If there is a member who does not vote within the timeframe, I will select the `ADMIN ACTION` vote for them.

**This is an anonymous vote.** Everyone with a unique link will be able to see whether or not you have voted, but they **will not see** who you voted for. Nobody will be able to see that, except for you if you revisit your unique voting URL.

**Comment by mhdawson** [comment]
@bnb thanks ! 

As a reminder the posts were we outline why we each are running:

* Rich - https://github.com/nodejs/TSC/issues/521#issuecomment-382055945
* Michael -https://github.com/nodejs/TSC/issues/521#issuecomment-383249095

**Comment by bnb** [comment]
Looks like the number of unique URLs (18) being generated by the voting platform is causing some troubles for it. Investigating and will keep y'all up to date 👍

**Comment by bnb** [comment]
Since the site we normally use is not working properly at present, we're going to go with a different anonymous voting site that has been used by the Node.js Foundation in the past.

Updated instructions:

You'll recieve an email from `no_reply@adoodle.org` that will have a voting link. Please follow the voting link and vote accordingly. This is **still an anonymous vote**. Again, don't share your links with anyone.

**Comment by bnb** [comment]
Voting on the TSC Chairperson role has concluded as of 12:00 ET today, and has ended in a 50/50 split.

As far as I'm aware, the TSC doesn't have prior context on how a tie will be handled. Happy to facilitate further, as an impartial observer, if needed.

![image](https://user-images.githubusercontent.com/502396/40010871-0f539ed0-5774-11e8-8519-c16af9f55770.png)

**Comment by jasnell** [comment]
The @nodejs/tsc doesn't have a process for this currently. On the foundation side, ties are settled by coin toss. If both @mhdawson and @Trott are going to be present at the upcoming Collaborator Summit, may I suggest that as a possible resolution?

**Comment by Trott** [comment]
I had already told @mhdawson and @bnb  privately that if there is a tie vote, I will concede to @mhdawson on the grounds that in a deadlock, _status quo_ wins out. Not sure how this would work if there wasn't an incumbent, but since there is: Congratulations, @mhdawson!

**Comment by mhdawson** [comment]
@Trott thanks for being so gracious.

**Comment by Trott** [comment]
This issue can be closed now, I think. Obviously, re-open if I'm mistaken.
